pkgs/development/libraries/wavpack/default.nix

```diff
@@ -1,21 +1,22 @@
-{ stdenv, fetchurl, libiconv }:
+{ lib, stdenv, fetchurl, libiconv }:
 
 stdenv.mkDerivation rec {
   pname = "wavpack";
-  version = "5.3.0";
+  version = "5.4.0";
 
   enableParallelBuilding = true;
 
   buildInputs = stdenv.lib.optional stdenv.isDarwin libiconv;
 
   src = fetchurl {
-    url = "http://www.wavpack.com/${pname}-${version}.tar.bz2";
-    sha256 = "00baiag7rlkzc6545dqdp4p5sr7xc3n97n7qdkgx58c544x0pw5n";
+    url = "https://www.wavpack.com/${pname}-${version}.tar.bz2";
+    sha256 = "0d60r8bfvnm6j8ckajjfrcvrix5ac6yvn28vbq062bd7z7fac5h7";
   };
```

Comment:
Could we switch to `fetchFromGitHub` instead? Asking this because this would make it easier to bump using automated tools like `nix-update` for example.

Reply:
Just switching to fetchFromGitHub did not seem to work.

Reply:
Strange, it should. But oh well, it is fine the way it is too.

Reply:
The files on wavpack.com are release tarballs, whereas github is a source tarball. To build it from github, you'd need to use `autoreconfHook` or run `autogen.sh` in `preConfigure`.